src/wattsi.pas

```diff
@@ -45,6 +45,7 @@
 
 var
    Quiet: Boolean = false;
+   Version: Byte = (*$I version.inc *); // unsigned integer from 0 .. 255
```

Comment:
Right now we have 33 commits. I'm not sure we'll ever reach 255, but this still makes me nervous.